sagemaker-inference-deployment-guardrails/Update-SageMaker-Inference-endpoint-using-linear-traffic-shifting.ipynb

```diff
@@ -70,7 +70,15 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    if metrics_epc_1 is not None:
</pre><p>CI error shows:</p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
Exception encountered at "In [21]":
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
&lt;ipython-input-21-5c0afce1ed23&gt; in &lt;module&gt;
      3 )
      4 
----&gt; 5 if metrics_epc_1 is not None:
      6         metrics_all = invocation_metrics.join([metrics_epc_1], how="outer")
      7 

NameError: name 'metrics_epc_1' is not defined
</pre><p>Looks like the definition of <code>metrics_epc_1</code>  and <code>metrics_epc_2</code> are being deleted, which causes the failure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>